CHANGELOG.md

```diff
@@ -26,10 +26,12 @@ All notable changes to this project will be documented in this file.
 - Document that Spark Connect doesn't integrate with the history server ([#559])
 - test: Bump to Vector `0.46.1` ([#565]).
 - Use versioned common structs ([#572]).
+- Change the label `app.kubernetes.io/name` for Spark history and connect objects to use `spark-hhistory` and `spark-connect` instead of `spark-k8s` ([#573]).
```

Comment:
```suggestion
- Change the label `app.kubernetes.io/name` for Spark history and connect objects to use `spark-history` and `spark-connect` instead of `spark-k8s` ([#573]).
```